Analysis/Tasks/PWGHF/HFLcK0sPCandidateSelector.cxx

```diff
@@ -0,0 +1,400 @@
+      return false; // mass of the Lambda
+    }
+
+    if (std::abs(InvMassGamma(hfCandCascade) - RecoDecay::getMassPDG(kGamma)) < cuts[ptBin][2]) {
+      LOG(DEBUG) << "mass gamma cut failed: from v0 in cascade, gamma --> " << InvMassGamma(hfCandCascade) << ", cut --> " << cuts[ptBin][2];
+      return false; // mass of the Gamma
+    }
+
+    if (hfCandCascade.ptProng0() < cuts[ptBin][3]) {
+      LOG(DEBUG) << "bach pt cut failed, from cascade --> " << hfCandCascade.ptProng0() << " , cut --> " << cuts[ptBin][3];
+      return false; // pt of the p
+    }
+
+    if (hfCandCascade.ptV0Pos() < cuts[ptBin][4]) {
+      LOG(DEBUG) << "v0 pos daugh pt cut failed, from cascade --> " << hfCandCascade.ptV0Pos() << ", cut --> " << cuts[ptBin][4];
+      return false; // pt of the K0
+    }
+
+    if (hfCandCascade.ptV0Neg() < cuts[ptBin][4]) {
+      LOG(DEBUG) << "v0 neg daugh pt cut failed, from cascade --> " << hfCandCascade.ptV0Neg() << ", cut --> " << cuts[ptBin][4];
+      return false; // pt of the K0
+    }
+
+    if (hfCandCascade.pt() < cuts[ptBin][5]) {
+      LOG(DEBUG) << "cand pt cut failed, from cascade --> " << hfCandCascade.pt() << ", cut --> " << cuts[ptBin][5];
+      return false; // pt of the Lc
+    }
+
+    if (std::abs(hfCandCascade.impactParameter0()) > cuts[ptBin][6]) {
+      LOG(DEBUG) << "d0 bach cut failed, in cascade --> " << hfCandCascade.impactParameter0() << ", cut --> " << cuts[ptBin][6];
+      return false; // d0 of the bachelor
+    }
+
+    /*
+    if ((std::abs(hfCandCascade.dcapostopv()) > cuts[ptBin][7]) || (std::abs(hfCandCascade.dcanegtopv()) > cuts[ptBin][7])) {
+      LOG(DEBUG) << "v0 daugh cut failed, positive v0 daugh --> " << hfCandCascade.dcapostopv() << ", negative v0 daugh --> " << hfCandCascade.dcanegtopv() << " , cut --> " << cuts[ptBin][7];
+      return false; // d0 of the K0s daughters
+    }
+    */
+
+    if (std::abs(hfCandCascade.impactParameter1()) > cuts[ptBin][7]) {
+      LOG(DEBUG) << "d0 v0 cut failed, in cascade --> " << hfCandCascade.impactParameter1() << ", cut --> " << cuts[ptBin][7];
+      return false; // d0 of the v0
+    }
+
+    return true;
+  }
+
+  /// Check if track is ok for TPC PID
+  /// \param track is the track
+  /// \note function to be expanded
+  /// \return true if track is ok for TPC PID
+  template <typename T>
+  bool validTPCPID(const T& track)
+  {
+    if (track.pt() < d_pidTPCMinPt || track.pt() >= d_pidTPCMaxPt) {
+      LOG(DEBUG) << "Bachelor pt is " << track.pt() << ", we trust TPC PID in [" << d_pidTPCMinPt << ", " << d_pidTPCMaxPt << "]";
+      return false;
+    }
+    return true;
+  }
+
+  /// Check if we will use TPC PID
+  /// \param track is the track
+  /// \note function to be expanded
+  /// \return true if track is ok for TPC PID
+  template <typename T>
+  bool applyTPCPID(const T& track)
+  {
+    if (TMath::Abs(track.pt()) < d_applyPidTPCMinPt) {
```

Comment:
Useless `TMath::Abs`

Reply:
I had not seen this comment before. fixed now.